publications-2007.bib

```diff
@@ -229,6 +229,17 @@ @PhDThesis{2007:nigro:discontinuous
   year    = 2007
 }
 
+@Article{2007:nomura.sato.ea:multi-disciplinary,
+  author  = { Nomura, Tsuyoshi and Sato, Kazuo and Nishiwaki, Shinji and Yoshimura, Masataka },
```

Comment:
I'm not a fan of the spaces after and before the parenthesis. Primarily because I don't know how bibtex deals with them. Will they be ignored? Will they be treated as non-breaking spaces? I don't know

Reply:
Let's not worry about this. This is how it comes out of the script, let's not spend too much work on these issues. If it renders poorly, we can always go back and change it again later, but I'm pretty sure that it will render the exact same way with or without the spaces.